Automattic/wp-calypso
# Stats: Add basic presentation logic for panel

Related to:

- https://github.com/Automattic/red-team/issues/154
- https://github.com/Automattic/red-team/issues/164

## Proposed Changes

Adds basic behaviours for the floating feedback panel.

1. Checks with Notices API to determine if panel should be displayed.
2. Introduces a short delay so the panel pops up after the page loads.
3. Updates hibernation value on server if panel is dismissed.

Note: No longer using local storage to store timestamps for logic. Also no longer attempting to track feedback submission status. That should probably happen directly in the modal instead of here.

## Why are these changes being made?


The floating panel needs some rules in place to govern how often it is presented to the user or it will quickly become annoying/frustrating for the user.

## Testing Instructions



* Launch the Calypso Live site.
* Visit the Traffic page.
* Enable the feature flag: `?flags=stats/user-feedback`
* Confirm the floating panel is visible at the bottom of the window (not the page).
* Dismiss the panel and reload the page.
* Confirm the panel is not shown again.
* Wait 30 seconds and reload the page (or change to the Insights tab and back).
* Confirm the panel is shown again.

## Pre-merge Checklist
